proposals/NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md

```diff
@@ -0,0 +1,68 @@
+start 10 sec timer right after assigning HMI level different from NONE for registered app for waiting SetGlobalProperties_request from mobile app. SDL must start this 10 sec timer only once in case of app registration OR after HMILevel resumption OR activation after registration.
+3. In case mobile app sends SetGlobalProperties_request during 10 sec timer with: valid <helpPrompt> and <vrHelp> params, other valid params related to this RPC.
+SDL must transfer TTS.SetGlobalProperties (<helpPrompts>, params) with adding period of silence between each command "helpPrompt" to HMI transfer UI.SetGlobalProperties (<vrHelp, params>) to HMI respond with <resultCode_received_from_HMI> to mobile app.
+4. In case SDL already transfers UI/TTS.SetGlobalProperties with <vrHelp> and <helpPrompt> received from mobile app to HMI  and 10 sec timer is NOT expired yet and mobile app sends AddCommand and/or DeleteCommand requests to SDL.
+SDL must update internal list with new values of "vrHelp" and "helpPrompt" params ONLY after successfull response from HMI.
+SDL must NOT send updated values of "vrHelp" and "helpPrompt" via TTS/UI.SetGlobalProperties requests to HMI.
+5. In case mobile app does NOT send SetGlobalProperties with <vrHelp> and <helpPrompt> to SDL during 10 sec timer and SDL already sends by itself UI/TTS.SetGlobalProperties with values of <vrHelp> and <helpPrompt> to HMI and mobile app sends AddCommand and/or DeleteCommand requests to SDL.
+SDL must update internal list with new values of "vrHelp" and "helpPrompt" params ONLY after successfull response from HMI send updated values of "vrHelp" and "helpPrompt" via TTS UI.SetGlobalProperties to HMI till mobile app sends SetGlobalProperties request with valid <vrHelp> and <helpPrompt> params to SDL.
+6. In case mobile app has NO registered AddCommands and/or DeleteCommands requests (resumed within data resumption process).
+SDL must use current appName as default value for "vrHelp" parameter, retrieve value of "helpPrompt" from .ini file ([GLOBAL PROPERTIES] section -> "HelpPrompt" param).
+7. In case mobile app sends SetGlobalProperties_request during 10 sec timer without <helpPrompt> and/or <vrHelp> params and with other valid params related to this RPC and this mobile app has successfully registered VR.AddCommands and/or VR.DeleteCommands requests (resumed during data resumption).
+SDL must provide the value of <helpPrompt> and/or <vrHelp> from internal list based on registered VR.AddCommands and VR.DeleteCommands requests to HMI.
+8. In case mobile app does NOT send SetGlobalProperties with <vrHelp> and <helpPrompt> to SDL during 10 sec timer and SDL already sends by itself UI/TTS.SetGlobalProperties with values of <vrHelp> and <helpPrompt> to HMI  and SDL does NOT receive response from HMI at least to one TTS/UI.SetGlobalProperties during <DefaultTimeout> (the value defined at .ini file).
+SDL must log corresponding error internally continue work as assigned (due to existing requirements).
```

Comment:
@GetmanetsIrina 
log corresponding error internally **and** continue work as expected according to existing requirements

Reply:
@GetmanetsIrina , suggest to remove entire sentence in ()

Reply:
Changed locally.